ycmd/completers/language_server/lsapi.py

```diff
@@ -156,6 +156,12 @@ def BuildTextDocumentPositionParams( request_data ):
   }
 
 
+def References( request_id, request_data ):
+  request = BuildTextDocumentPositionParams( request_data )
+  request[ 'includeDeclaration' ] = True
```

Comment:
as mentioned, needs to be `request[ 'context' ] = { 'includeDeclaration': True }`

Reply:
Done.